web/src/index.js

```diff
@@ -23,56 +25,80 @@ const registerServiceWorker = async () => {
     }
 };
 
-const startApplication = () => {
+const startApplication = async () => {
 
-    function createIframe(src, script) {
+    function createIframe(src, scriptFactory) {
         const iframe = document.createElement('iframe');
         iframe.setAttribute('src', src); // Initialize with an empty page
         iframe.setAttribute('width', '500');
         iframe.setAttribute('height', '300');
         document.body.appendChild(iframe);
 
-        iframe.addEventListener('load', function () {
+        const rand = (Math.random() + 1).toString(36).substring(7);
+        const scriptContent = scriptFactory(rand);
+
+        function injectScript() {
             // Access the iframe's document object
             const iframeDocument = iframe.contentDocument || iframe.contentWindow.document;
 
             // Create a new script element
             const scriptElement = iframeDocument.createElement('script');
 
             // Set the script's content with your custom code
-            scriptElement.textContent = script;
+            scriptElement.textContent = scriptContent;
 
             // Append the script to the iframe's document
             iframeDocument.body.appendChild(scriptElement);
+        }
+
+        // injectScript();
+
+        iframe.addEventListener('load', function () {
+            injectScript();
```

Comment:
wanted to fiddle with injecting scripts at different time / spamming the injection